### ISSUE TITLE
Same dataset

### ISSUE THREAD

**Issue by jdavidd** [Jeetu95/Brain-Tumor-Segmentation#1]
Hello, I'm using the same dataset for my final project. For 3 weeks I am trying to finetune my network to obtain something good but no result. I am using FNC8 with vgg encoder.
What I want to ask, if you have some time, I can give you my notebook wich run in colab, and if i can inspire from your work using your Unet arhitecture?
Sorry, it isnt an issue, but i write this, maybe I can talk to you somehow.

**Comment by Jeetu95** [comment]
Are you doing the same work as segmentation ???

**Comment by jdavidd** [comment]
Hello, yes I am trying to segmentate the image with FCN8s arhitecture, with VGG encoder. This is the notebook with details https://colab.research.google.com/drive/1gDPlKSDiY3Zj-0uk04e9xTDClURqImDe.
We can talk on gmail jitcamirel@gmail.com?
The major problem is i am trying with crossentropy loss function, train loss and valid loss is decreasing but accuracy is very low close to 0.
